src/main/webapp/register.jsp

```diff
@@ -223,11 +217,16 @@ if(ses.getAttribute("errorMessage") != null)
 		var theEmailAgain = $("#userAddressCnf").val();
 		//Validation
 		var theError = "";
+		let elasticEmailPattern = /(^[^@]*[^@])([@])(elastic\.co)$/;

```

Comment:
We don't need capturing groups since we don't need them and then make it case insensitive also ?
```suggestion
		let elasticEmailPattern = /^[^@]*[^@][@]elastic\.co$/i;

```